astroquery/alma/core.py

```diff
@@ -567,7 +566,7 @@ def get_data_info(self, uids, expand_tarfiles=False,
         """
         if uids is None:
             raise AttributeError('UIDs required')
-        if isinstance(uids, six.string_types + (np.bytes_,)):
+        if isinstance(uids, (str, np.bytes_)):
```

Comment:
use the built-in Python `bytes` instead
```suggestion
        if isinstance(uids, (str, bytes)):
```